### DISCUSSION TITLE
Automatische Umwandlung Flicker->QR

### DISCUSSION THREAD

**Discussion by Fireobirdy** [willuhn/hibiscus#136] [Entwickler-Diskussion]
Liebes Team,

ReinerSCT bietet ein Tool an, Flicker-Codes auf dem Bildschirm zu erkennen und in QR-Codes umzuwandeln. Ich würde einem Tool welches in Hibiscus integriert ist mehr vertrauen, als einem externen Tool, welches permanent den Bildschirm scannt. Daher wünsche ich mir(es ist ja bald Weihnachten) eine Funktion oder ein Plug-In, dass die Umwandlung ermöglicht, damit ich neuere Lesegeräte nutzen kann.

**Comment by willuhn** [comment]
Hibiscus unterstützt das bereits seit Version 2.10.5 (siehe https://github.com/willuhn/hibiscus/blob/master/build/ChangeLog#L56). In der Detailansicht des PIN/TAN-Bankzugangs wird hierzu eine Option "[x] Flickercode in QR-Code umwandeln" angezeigt, sobald das TAN-Verfahren als Flicker-Code-Verfahren erkannt wurde.

**Reply by Fireobirdy** [reply]
Vielen Dank. Hatte an der falschen Stelle gesucht. Kann gelöscht werden.

**Reply by willuhn** [reply]
Danke für die Rückmeldung. Die Diskussion muss nicht gelöscht werden - wenn andere User die selbe Frage haben, können Sie die Antwort hier ja per Google-Suche finden.
